addon/mixins/colorable.js

```diff
@@ -90,7 +90,7 @@ export default Ember.Mixin.create({
       var colorRange = _this.get('getColorRange')(seedColor);
       var colorScale = _this.get('getColorScale')(seedColor);
       if (numColorSeries === 1) {
-        return colorRange[0];
+        return colorRange[0].toString();
```

Comment:
I'd add a comment above the function on L88 saying this will return a HEX string